djimaging/tables/response/orientation.py

```diff
@@ -129,25 +132,47 @@ def get_si(dir_component, dirs, per):
     return index, direction
 
 
-def compute_null_dist(rep_dir_resps, dirs, per, iters=1000):
+def test_tuning(dirs, counts, per, iters=1000):
```

Comment:
This is now a direct translation of the original matlab function they were using to do this:
```{matlab}
function [p, q, qdistr] = testTuning(dirs, counts, per)
iter = 1000;
[M, N] = size(counts);
k = dirs(:);% / 180 * pi;
v = exp(per* 1i * k) / sqrt(N);
q = abs(mean(counts, 1) * v);
qdistr = zeros(iter, 1);
for j = 1 : iter
    r = randperm(M * N);
    counts(:) = counts(r);
    qdistr(j) = abs(mean(counts, 1) * v);
end
p = mean(qdistr > q);
```

Reply:
the main difference seems to be the normalization after shuffling